code/modules/ordeals/misc_ordeals.dm

```diff
@@ -0,0 +1,33 @@
+// Black Fixer Dusk
+/datum/ordeal/black_dusk
+	name = "Dusk of Black"
```

Comment:
I know that right now it's the only fixer, but I'd rather we don't name it after their colors.
Just name it the Dusk of White; Later on we could make it spawn random type of a fixer.

Reply:
And move it to the white_ordeals file, yeah.

Reply:
We do have an issue with white midnight later, We could have a midnight with all 4 fixers but it would require changing claw